### ISSUE TITLE
Query Cache: include system collections

### ISSUE THREAD

**Issue by Simran-B** [arangodb/docs#91]
Docs might not be clear enough about this option.

---

For the query cache there is a setting that allows inclusion of "system collections"

- Does "system collections" mean the collections whose name start with a `_` *in the current database* or collections in the `_system` database?
- What are the (known) implications of setting the setting to include system collections in query cache to `true`?

-> System collections here means any collection whose name starts with an `_`, regardless of from which database it is.

is this caching global to the arangodb instance then?
-> Yes
Normally system collections can be ignored in the query cache. For example, buffering any query results for queries on the `_statistics` collection normally doesn't make any sense, as new data will be added regularly. So each (internal) query on `_statistics` will see a different state of data anyway.
It could be cached, but it would be useless and simply waste memory.
`_users` are cached in memory anyway, so buffering AQL query results on `_users` in the query cache again doesn't make much sense.
In short: data from system collections could be cached, but normally it is not worth doing it.

there's no "security" considerations or suchlike?
-> No, Permissions/privileges are still checked even if the query cache is turned on
